### DISCUSSION TITLE
Merge PullRequest and Actions should display the job information

### DISCUSSION THREAD

**Discussion by alexmbra** [community/community#124167] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I have a yml file that has an action for the start of a pull request and another for the merge the pull request and it works fine. When the pull request starts, it display info about the associated job. 

But when I click on the merge button, it starts to run this job:
deploy:
    name: Deploy on PR Merge
    runs-on: ubuntu-latest
    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.ref == 'refs/heads/master'
    ...

  It does not display information of the running job. Instead shows this message: "Pull request successfully merged and closed". 
 It should display the info about the running job and only show that message after the end of that job.

**Comment by abdemeh** [comment]
To achieve the behavior you're looking for in your GitHub Actions workflow, you need to ensure that the job's status and log information are correctly displayed when a pull request is merged,
There are a few potential reasons why your current setup may not be showing the job information as expected:
- Event and Context Check: Ensure that the event and context checks are correctly defined so that the workflow runs at the right time and the job's status is displayed.
- Workflow Structure: Ensure that the workflow file is structured in a way that GitHub Actions can interpret and run it correctly.
- Permissions and Access: Verify that the necessary permissions are set up to allow the workflow to access and display job information.
